src/eduid/webapp/idp/mischttp.py

```diff
@@ -160,6 +160,9 @@ def set_sso_cookie(value: str, response: WerkzeugResponse) -> WerkzeugResponse:
     :param value: The value to assign to the cookie
     :param response: Flask response object
     """
+    # local import to avoid import loop
+    from eduid.webapp.idp.app import current_idp_app as current_app
```

Comment:
It should be trivial to make set_sso_cookie to take key: str as first argument and current_app.conf.sso_cookie as "cookie_config" so we can let the views care about current_app.